monkey/tests/unit_tests/infection_monkey/master/test_propagator.py

```diff
@@ -24,8 +24,7 @@ def __init__(self):
             pass
 
         def build_target_host(self, network_address: NetworkAddress) -> TargetHost:
```

Comment:
This function is no longer needed. Whatever calls this can just construct `TargetHost` directly.

Reply:
Opening another PR for this soon

Reply:
That will be reworked when we remove the `TargetHostFactory`